storage/pebble/registers.go

```diff
@@ -1,22 +1,46 @@
 package pebble
 
 import (
+	"encoding/binary"
 	"fmt"
 
 	"github.com/cockroachdb/pebble"
 
 	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
 )
 
-// library that implements pebble storage for registers
+// Registers library that implements pebble storage for registers
+// given a pebble instance with root block and root height populated
 type Registers struct {
-	db *pebble.DB
+	db           *pebble.DB
+	firstHeight  uint64
+	latestHeight uint64
 }
 
+// NewRegisters takes a populated pebble instance with LatestHeight and FirstHeight set.
+// Will fail if they those two keys are unavailable as it implies a corrupted or uninitialized state
```

Comment:
Who does initialize it?

Reply:
the `Bootstrap` library here: https://github.com/onflow/flow-go/pull/4727 I'll update that with the stub function to unblock you